### PR TITLE
Add graphene

### PR DESCRIPTION
**Pull request by achadwick**
Seemingly now a build dependency of git GTK.

Introspection isn't working yet.

### CODE REVIEW

File: mingw-w64-graphene/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
+pkgver=${_realver}
+pkgrel=1
+arch=('any')
+pkgdesc="A thin layer of types for graphic libraries (mingw-w64)"
+depends=(
+    "${MINGW_PACKAGE_PREFIX}-glib2"
+    "${MINGW_PACKAGE_PREFIX}-gettext"
+)
+makedepends=(
```
Comment (Alexpux) [inline mingw-w64-graphene/PKGBUILD]: Remove all this from makedepends. This is common dependencies for all packages building. Here we write only specific for this package dependencies.

File: mingw-w64-graphene/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    "libtool"
+    "pkg-config"
+    "gtk-doc"
+    # "${MINGW_PACKAGE_PREFIX}-gobject-introspection"
+)
+#options=('!strip' 'debug' 'staticlibs')
+options=('strip' '!debug')
+license=("MIT")
+url="http://ebassi.github.com/graphene"
+source=("https://github.com/ebassi/graphene/archive/${_realver}.tar.gz")
```
Comment (Alexpux) [inline mingw-w64-graphene/PKGBUILD]: Change source entry to download tarsal into named  file. See other packages.

File: mingw-w64-graphene/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+prepare() {
+    cd "${srcdir}"/${_realname}-${_realver}
+}
+
+build() {
+    [[ -d build-${MINGW_CHOST} ]] && rm -rf build-${MINGW_CHOST}
+    cp -a "${srcdir}/${_realname}-${_realver}" build-${MINGW_CHOST}
+    cd build-${MINGW_CHOST}
+    ./autogen.sh \
```
Comment (Alexpux) [inline mingw-w64-graphene/PKGBUILD]: I think proper is to run autogen in prepare and then run configure in build.

File: mingw-w64-graphene/PKGBUILD (hunk 1 above)
Comment (achadwick) [inline mingw-w64-graphene/PKGBUILD]: Okay. I dropped gtk-doc too, since it's not used - though omitting it has broken GTK in the past.

**Pull request merged**